stdlib/public/runtime/SwiftObject.mm

```diff
@@ -1365,7 +1365,13 @@ static bool usesNativeSwiftReferenceCounting_nonNull(
 void swift_objc_swift3ImplicitObjCEntrypoint(id self, SEL selector,
                                              const char *filename,
                                              size_t filenameLength,
-                                             size_t line, size_t column) {
+                                             size_t line, size_t column,
+                                             std::atomic<bool> *didLog) {
```

Comment:
Is `std::atomic<bool>` guaranteed to be layout-compatible with `_Atomic(_Bool)`? I couldn't find such a guarantee in the C++ standard, although libc++ certainly seems to provide it.